components/ws-daemon/pkg/cgroup/plugin_iolimit_v2.go

```diff
@@ -61,38 +64,52 @@ func (c *IOLimiterV2) Apply(ctx context.Context, basePath, cgroupPath string) er
 	return nil
 }
 
-func (c *IOLimiterV2) writeIOMax(loc string) error {
-	iostat, err := os.ReadFile(filepath.Join(string(loc), "io.stat"))
+func (c *IOLimiterV2) writeIOMax(cgroupPath string) error {
+	iostat, err := os.ReadFile(filepath.Join(string(cgroupPath), "io.stat"))
 	if os.IsNotExist(err) {
+		log.Error("Pod is gone")
 		// cgroup gone is ok due to the dispatch/container race
 		return nil
 	}
+
 	if err != nil {
 		return err
 	}
+
+	var classesToLimit = []string{"8", "9"}
```

Comment:
Since these numbers is too magic, how about add some comments to make readable for humans.
```suggestion
   // 8: block	SCSI disk devices (0-15)
   // 9: char	SCSI tape devices
	var classesToLimit = []string{"8", "9"}
```

Reply:
done